pwiz_tools/Skyline/Model/Lib/BiblioSpecLite.cs

```diff
@@ -995,7 +992,9 @@ private bool Load(ILoadMonitor loader, IProgressStatus status, bool cached, out
                     }
                     catch (Exception e)
                     {
-                        failureException = new Exception(FormatErrorMessage(e), e);
+                        var failureException = new Exception(FormatErrorMessage(e), e);

```

Comment:
I think you could get rid of the "var failureException" and simply put all the new Exception() constructors inside the status.ChangeErrorException() calls.